RFS/src/main/java/com/rfs/version_es_6_8/IndexMetadataData_ES_6_8.java

```diff
@@ -30,7 +32,7 @@ public String getId() {
     }
 
     @Override
-    public ObjectNode getMappings() {
+    public JsonNode getMappings() {
```

Comment:
Just curious - do you think it makes sense to change all our ObjectNodes to JsonNodes at some point?  JsonNode is more flexible, as exhibited here, but the type casting it will require could get annoying.  I'm really not sure what the right answer is.  Maybe it's just exactly what you did - use ObjectNode wherever possible and fall back to JsonNode when we must.

Reply:
I think JsonNode is likely a better candidate - not sure if/when it will make sense to 'cut everything over', good thing to watch for oppertunities